reader_test.go

```diff
@@ -18,39 +17,78 @@ var _ = Describe("Reader", func() {
 	var obj *bfs.Object
 	var ctx = context.Background()
 
-	BeforeEach(func() {
-		obj = bfs.NewInMemObject("path/to/file.json")
-		Expect(writeMulti(obj, 3, time.Time{})).To(Succeed())
+	Describe("NewReader", func() {
+		BeforeEach(func() {
+			obj = bfs.NewInMemObject("path/to/file.json")
+			Expect(writeMulti(obj, 3, time.Time{})).To(Succeed())
 
-		var err error
-		subject, err = feedx.NewReader(ctx, obj, nil)
-		Expect(err).NotTo(HaveOccurred())
-	})
+			var err error
+			subject, err = feedx.NewReader(ctx, obj, nil)
+			Expect(err).NotTo(HaveOccurred())
+		})
 
-	AfterEach(func() {
-		Expect(subject.Close()).To(Succeed())
-	})
+		AfterEach(func() {
+			Expect(subject.Close()).To(Succeed())
+		})
 
-	It("reads", func() {
-		data, err := ioutil.ReadAll(subject)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(len(data)).To(BeNumerically("~", 110, 20))
-		Expect(subject.NumRead()).To(Equal(0))
-	})
+		It("reads", func() {
+			data, err := io.ReadAll(subject)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(data)).To(BeNumerically("~", 110, 5))
```

Comment:
Minor: Can it REALLY fluctuate (size)? I think this comes from `writeMulti(obj, 3, time.Time{})` -- all these zero-times should always have the same marshalled size.